src/main.cpp

```diff
@@ -487,7 +487,7 @@ int main(int argc, char* argv[])
         QFile::link(homeDirectory, homeLink);
     }
 #endif
-    app->setStyle(new AltFocusMenuBarDisable);
```

Comment:
Do we know why this was added in the PR which added the dark theme? Should this be making its way in somewhere else to achieve the same effect?

Reply:
Well it was added by #4984 by @Edru2 on 2021/10/02 whereas the other bits came from #5574 by @vadi2 on 2021/11/15. I can only assume that the latter wasn't fully aware of the former - and homed in on my (already not working at that point) code and only thought that was where his code ought to go...

As it is that bit modifies whatever style factory was going to be use so as to ***prevent*** the following:

`Menu bars items are navigable by pressing Alt, followed by using the arrow keys to select the desired item.`

given that it is now/also applied in the `mudlet` constructor by Vadi's code which is initiated in the next line I cannot see that there is any functional change in NOT having it there...

Reply:
If it's repeated elsewhere and I just didn't see it due to reviewing diff that's different then. May still prefer one of the more experienced c++ folks, or one of the above referenced folks comments on it. A little out of my depth.

But can confirm functional testing works.

Reply:
Pressing Alt button caused input bar to lose focus, therefore it was added originally by myself.

Reply:
But that is hardly an issue *before* any profiles are even loaded - so I think it is safe to leave it to be done in the `mudlet` class constructor.. :wink:

Reply:
C++ checks out for me. @demonnic could you confirm functional testing was for the Alt key causing a loss of focus?

Reply:
I can confirm the alt button does not cause it to lose focus on this testing version.

I do not see the keyboard shortcuts next to the menu items any more though.

4.14.1
![image](https://user-images.githubusercontent.com/3660/144260594-7ea9d0c8-907c-4ce9-b566-08323b350c3c.png)

this PR
![image](https://user-images.githubusercontent.com/3660/144260685-cd6a6ff4-48bb-47e2-9762-db294d333112.png)


They work, but they aren't called out. In contrast to current release where they're called out, until they stop working and then they are no longer listed. Not sure if that's related.

Reply:
can verify this happens in the PTB as well, which I had not had time to check earlier. So not introduced as part of this PR.